src/misc/programs.cpp

```diff
@@ -62,6 +62,25 @@ using comdata_t = std::vector<uint8_t>;
 static std::vector<comdata_t> internal_progs_comdata;
 static std::vector<PROGRAMS_Creator> internal_progs;
 
+static uint8_t last_written_character = '\n';
+constexpr int WRITEOUT_BUFSIZE = 2048;
+
+static void WriteToStdout(std::string_view output)
```

Comment:
Our convention is lower_snake_case for static functions, lambdss, and small public helper functions.